ivre/tools/ipdata.py

```diff
@@ -92,21 +94,23 @@ def my_parse_args():
                 ivre.db.db.data.ensure_indexes()
     if args.download:
         ivre.geoiputils.download_all(verbose=not args.quiet)
+    dbtofeed = {}
+    if not args.dont_feed_ipdata_cols:
+        dbtofeed["feedipdata"] = [ ivre.db.db.passive ]
+        if args.update_nmap_db:
```

Comment:
You do mean an "update-passive-db" option, or do you want to keep this default behavior?
In the latter case, I would probably rename the nmap option to "--no-update-nmap-db", since it would be confusing to have one non-negated option for one db, and a negated one for the other.